docs/usage.rst

```diff
@@ -72,30 +133,52 @@ command:
 
 .. code-block:: console
 
-   $ magic-folder --node-directory=alice list
+   $ magic-folder --config ./foo list
    This client has the following magic-folders:
-     default: /home/alice/Work/Magic
-       magic: /home/alice/Documents/Shared
+   example:
+       location: /home/alice/Documents
+      stash-dir: /home/alice/foo/example/stash
+         author: alice-laptop (public_key: KSYPPXN3HTCSEJC56RRYXDEO2TZX5LO743Q3E2M7NA7UP2W3OK2A====)
+        updates: every 60s
+
+To get JSON output, pass ``--json``.  You can include sensitive secret
+information by passing ``--include-secret-information`` flag. Someone
+who obtains this information can impersonate this device and
+participate as it in the magic folder (if they also gain access to the
+Tahoe-LAFS Grid being used).
 
 
 Inviting Participant Devices
 ----------------------------
 
 A new participant device is invited to collaborate on a magic folder
-using the ``magic-folder invite`` command:
-
+using the ``magic-folder invite`` command. This produces an "invite
+code" which is a one-time code. This code should be communicated
+securely to the invitee. The code will allow the invitee's device to
+establish a connection to this device and exchange details. Thus, the
+code can only be used while this device is connected to the
+Internet. The code may only be used once, for a single invitee.
 
 .. code-block:: console
 
-   $ magic-folder --node-directory=alice invite magic: bob
+   $ magic-folder --config ./foo invite --name example
+
+An invitation code is created using an existing magic folder (``--name
+example`` above). The magic-folder identified must have been created on
+this device.
+
+Once the invitee runs ``magic-folder join`` (see below) the two
+devices will connect and exchange some information; this will complete
+the invitation. The "invite" command won't exit until the invitee has
+actually completed and will print out some details. If you pass
+``--no-wait`` then the command will exit immediately (although the
+invite will still be valid).
```

Comment:
Hmm, interesting .. I was thinking kind of the other way around (that most people would want to wait so that they can confirm the invite was received). I was thinking the "no wait" use-case would be for e.g. GridSync. To support "wait, or not" the actual HTTP API will have to be two-part anyway (i.e. "start invite", receive token ... "check status of invite").

Since the invite stuff isn't actually implemented in this PR I can put both option into the text and we can change the text when we do invites?

Reply:
in 18cec59 I add some words about the above